edit_course/templates/edit_course/edit_content.html

```diff
@@ -106,15 +107,16 @@ <h3 class="panel-title">{% trans "Course modules" %}</h3>
                 </td>
                 <td>
                     <a class="btn btn-default btn-xs" href="{{ lobject|editurl:'exercise' }}">
-                        <span class="glyphicon glyphicon-edit" aria-hidden="true"></span> {% trans "Edit exercise" %}
+                      <span class="glyphicon glyphicon-edit" aria-hidden="true"></span>
+                      {% if is_exercise %}{% trans "Edit exercise" %}{% else %}{% trans "Edit content chapter" %}{% endif %}
                     </a>
                     <a class="btn btn-default btn-xs" href="{{ lobject|removeurl:'exercise' }}">
                         <span class="glyphicon glyphicon-remove" aria-hidden="true"></span> {% trans "Remove" %}
                     </a>
                     <a class="btn btn-link btn-xs" href="{{ lobject|url }}">
-                    	{% trans "Open exercise" %}
+                    	{% if is_exercise %}{% trans "Open exercise" %}{% else %}{% trans "Open chapter" %}{% endif %}
                     </a>
-                    {% if lobject.is_submittable %}
+                    {% if is_exercise %}
```

Comment:
It looks like that `{% if lobject.is_submittable %}` is currently broken in production and it always evaluates to False. (The "view submissions" link is never visible.) This must also be fixed now. This used to work in the past, perhaps over a year ago.

We must find the code that really defines the value of `lobject.is_submittable` and hopefully it will be easy to fix it. Has the name of the attribute been changed to something else than `is_submittable`? Is the code that defines the value broken?